templates/pro/index.html

```diff
@@ -622,6 +622,20 @@ <h2>Enterprise</h2>
         </div>
       </div>
     </section>
+    <section class="p-section--shallow">
+      <div class="row">
+        <hr class="p-rule is-muted" />
+      </div>
+      <div class="row--50-50">
+        <div class="col">
+          <h2>Embedded Devices</h2>
+        </div>
+        <div class="col">
+          <p>Use Ubuntu&nbsp;Pro for your IoT fleet to easily manage all your package updates and guarantee security and compliance.</p>
+          <a href="/pro/devices" class="p-button--positive">Learn more</a>
```

Comment:
I think this button needs to just a default one
```suggestion
          <a href="/pro/devices" class="p-button">Learn more</a>
```